common/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/common/model/PrisonerTest.kt

```diff
@@ -0,0 +1,77 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.common.model
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+
+class PrisonerTest {
+  @Test
+  fun `all Prisoner fields are nullable`() {
+    assertDoesNotThrow {
+      jacksonObjectMapper().readValue("{}", Prisoner::class.java)
+    }
+  }
+
+  @Test
+  fun `all PrisonerAlias fields are nullable`() {
```

Comment:
this would be better to be parameterised, although would be even better to use recursive reflection to find out the objects, otherwise if someone adds another object then they'd need to remember to add it to this test class.